webquizappbackend/quizzes/views.py

```diff
@@ -142,27 +142,20 @@ def quiz_list(request):
         return JsonResponse({'message': '{} Quizzes were deleted successfully!'.format(count[0])}, status=status.HTTP_204_NO_CONTENT)
 
 @api_view(['GET', 'PUT', 'DELETE'])
-def quiz_detail(request, pk):
+def quiz_active(request, pk):
```

Comment:
Just wanted to make sure with testing that this didn't impact other features. I know we talked about this previously and it didn't appear to be used.